mpa/modules/models/classifiers/sam_classifier.py

```diff
@@ -252,7 +255,10 @@ def extract_feat(self, img):
         """Directly extract features from the backbone + neck
            Overriding for OpenVINO export with features
         """
-        x = self.featuremap = self.backbone(img)
+        x = self.backbone(img)
+        if torch.onnx.is_in_onnx_export:
```

Comment:
Is it typo? I think this line should be `torch.onnx.is_in_onnx_export()` , always True with current code.

Reply:
yes, I missed it. Fixed.